fabric-network/lib/gateway.js

```diff
@@ -45,7 +45,7 @@ const logger = require('./logger').getLogger('Gateway');
  * @typedef {Function} Gateway~TxEventHandlerFactory
  * @memberof module:fabric-network
  * @param {Transaction} transaction The transaction for which the handler should listen for commit events.
- * @param {module:fabric-network.Network} network The network on which this transaction is being submitted.
+ * @param {DefaultEventHandlerOptions} [options] Options for the event handler capability.
```

Comment:
I suspect the type link needs to be `module:fabric-network.DefaultEventHandlerOptions` for it to get the correct link in the built JSDoc. The `Transaction` link in the line above looks wrong too.